plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/messages.properties

```diff
@@ -4,3 +4,8 @@ NEW_SERVER_DIALOG_PORT=Server port:
 NEW_SERVER_DIALOG_INVALID_PORT_VALUE=Port must be between 0 and 65535.
 PORT_IN_USE=Port {0} is in use.
 PORT_OUT_OF_RANGE=Port must be between 0 and 65535.
+
+CREATE_APP_ENGINE_RUNTIME_WIZARD_DESCRIPTION=The App Engine Standard runtime requires the Google Cloud SDK
+CREATE_APP_ENGINE_RUNTIME_WIZARD_TITLE=App Engine Standard Runtime
+OPEN_CLOUD_SDK_PREFERNCE_BUTTON=Open the Cloud SDK Location preference page when wizard closes
```

Comment:
Just to clarify, it should change to:

Open the Cloud SDK location preference when the wizard closes